### PR TITLE
Fix Date(Number.POSITIVE_INFINITY) assertion fail in the Date object

### PR DESCRIPTION
**Pull request by szledan**
JerryScript-DCO-1.0-Signed-off-by: Szilard Ledan szledan.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by szledan** [comment]
Fix for #410

File: jerry-core/ecma/builtin-objects/ecma-builtin-date.cpp [hunk 1]
```diff
@@ -556,6 +556,11 @@ ecma_builtin_date_dispatch_construct (const ecma_value_t *arguments_list_p, /**<
 
   if (ecma_is_completion_value_empty (ret_value))
   {
+    if (!ecma_number_is_nan (*prim_value_num_p) && ecma_number_is_infinity (*prim_value_num_p))
```
Comment (ruben-ayrapetyan) [inline jerry-core/ecma/builtin-objects/ecma-builtin-date.cpp]: If a number is `Infinity`, it cannot be `NaN`.

Comment (galpeter) [inline jerry-core/ecma/builtin-objects/ecma-builtin-date.cpp]: But this tests that if it is **not** NaN and is infinity.

Also note that in the first line of the `ecma_number_is_infinity` there is an assert:
```
JERRY_ASSERT (!ecma_number_is_nan (num));
```

Comment (ruben-ayrapetyan) [inline jerry-core/ecma/builtin-objects/ecma-builtin-date.cpp]: @galpeter, I see. Thank you for explanation.

**Comment by ruben-ayrapetyan** [comment]
Looks good to me

**Comment by kkristof** [comment]
Looks good to me.

**Comment by galpeter** [comment]
Landed: https://github.com/Samsung/jerryscript/commit/bbfddea032fc622cef6279b308f5d77e135dd19b

**Pull request closed without merging**